tensorboard/components/tf_dashboard_common/tf-filterable-checkbox-dropdown.html

```diff
@@ -0,0 +1,176 @@
+<!--
+@license
+Copyright 2016 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<link rel="import" href="../iron-icon/iron-icon.html">
+<link rel="import" href="../paper-checkbox/paper-checkbox.html">
+<link rel="import" href="../paper-input/paper-input.html">
+<link rel="import" href="../paper-menu-button/paper-menu-button.html">
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../tf-imports/lodash.html">
+<link rel="import" href="run-color-style.html">
+<link rel="import" href="scrollbar-style.html">
+
+<!--
+tf-filterable-checkbox-dropdown creates a dropdown whose content is a list of
+checkboxes with a filter input at the top. The list is primarily used for
+multiple selection of items.
+
+Properties in:
+  - label: label for the dropdown trigger button.
+  - placeholder: placeholder in the dropdown trigger button.
+  - colorsCheckbox: whether to color the check boxes.
```

Comment:
Optional, but seems to me that "colorCheckboxes" or "useCheckboxColors" would make it clearer that this is a boolean value.

Reply:
Done.